README.md

```diff
@@ -0,0 +1,79 @@
+# Upcoming Movies Web App
+This is a MVP project to list upcoming movies integrate with [The Movie Database](https://themoviedb.org).
```

Comment:
Corrigir regência verbal:
This is a MVP project to list upcoming movies **integrated** with ...